app/models/course.rb

```diff
@@ -611,7 +611,52 @@ class Course < ActiveHash::Base
           ]
         }
       ]
-    }
+    }, {
+      id: 17,
+      name: 'Learn to Code - Go',
+      title: 'Learn Code with Go',
+      description: 'This is an introductory event. The focus will be on learning programming fundamentals using Go.',
+      levels: [
+        {
+          level: 1,
+          color: 'green',
```

Comment:
@dlsniper It probably doesn't make sense to have two levels with the same color. Can you change this one to blue?